app/controllers/calls_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CallsController < AdminController
-  load_and_authorize_resource find_by: :slug, except: :quantitative_case
+  # load_and_authorize_resource find_by: :id, except: :quantitative_case
```

Comment:
@Novsochetra please uncomment this line. It should be work fine.

Reply:
@sophalsun, I've updated it.